ui/app/templates/partials/secret-list/item.hbs

```diff
@@ -14,88 +14,80 @@
       <SecretLink
         @mode={{if item.isFolder "list" "show" }}
         @secret={{item.id}}
-        @class="has-text-black has-text-weight-semibold"
-      ><Icon
+        @class="has-text-black has-text-weight-semibold">
+        <Icon
           @glyph={{if item.isFolder 'folder-outline' 'file-outline' }}
-          @class="has-text-grey-light"
-        />{{if (eq item.id ' ') '(self)' (or item.keyWithoutParent item.id)}}
+          @class="has-text-grey-light"/>
+        {{if (eq item.id ' ') '(self)' (or item.keyWithoutParent item.id)}}
```

Comment:
This is annoying, but there needs to not be any space between this closing tag and the text.

This is what it was like before: 
![Screen Shot 2019-08-21 at 4 34 52 PM](https://user-images.githubusercontent.com/39469/63470320-f424ba00-c431-11e9-9a4d-a1eee1c79d53.png)

And with the extra line break, you get an underline between the icon and the word
![Screen Shot 2019-08-21 at 4 35 54 PM](https://user-images.githubusercontent.com/39469/63470321-f424ba00-c431-11e9-951b-a4f1d99b0cd5.png)

Reply:
heh this is a hangup of mine, or when it [only happens on hover](https://github.com/travis-ci/travis-web/pull/830) etc

Reply:
oy, that is frustrating. i'll fix it. thanks!